05-RemoveFile/README.md

```diff
@@ -2,6 +2,66 @@
 
 ## Introduction
 
+* Removing a file from repository is done in two steps.
+* First you remove the file in the staging area.
+* Next you have to commit your changes.
+* Directories are not tracked by _GIT_.
+  * Once a directory becomes empty it will be discarded.
+  * This can be a difference compared to TFS. If you would keep a folder
+    for some reason create a ```.gitattribute``` file inside.
```

Comment:
Also `.placeholder`, `.gitignore` and `.gitkeep` are a popular names. Basically I would just point out that it can be **any** file, not just `.gitattribute`.